easybuild/easyconfigs/p/Pillow/Pillow-6.2.2-foss-2020b-Python-2.7.18.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '6.2.2'
+# This is the last Pillow version to support Python 2.7.
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pillow.readthedocs.org/'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
Hmm, any reason why this isn't using the `GCCcore` toolchain, as is the case for existing (recent) `Pillow` easyconfigs?
cc @smoors

Reply:
The only reason is that this was based on earlier Pillow easyconfigs, being a build for Python 2.7.  I'd be happy to change this one to use the GCCcore toolchain, but I see that I cannot now edit the file - presumably because it has already been merged into the 'develop' branch.  Is there a way to reverse that merge, so that I can edit the PR?